### DISCUSSION TITLE
2.7.2 更新日志

### DISCUSSION THREAD

**Discussion by LTCatt** [Meloong-Git/PCL#3821] [讨论]
· 游戏内存管理优化
　　添加“启动游戏前进行内存优化”设置（#2187，@zkitefly）
　　修改内存参数以提高 MC 启动速度与性能（#1874，@ZZZank）
　　略微降低内存充足时为 MC 分配的内存量
· 窗口大小设置优化
　　将 [保持游戏默认] 和 [默认大小] 选项合并为 [默认] 选项
　　将默认窗口尺寸从 854×480 扩大到 875×540，以保证 UI 缩放不为小尺寸
　　修改部分文本描述
· 下载优化
　　允许在解压整合包的同时下载 Mod，以加快整合包安装速度
　　下载速度与进度等信息刷新得更为频繁，显示更加准确
　　减少多线程下载时在开始阶段的请求量
　　调整对 BMCLAPI 和 Modrinth 的请求频率，减少无用连接
· 追加更多 Fabric 给出崩溃解决方案的检测（#3699，by @zkitefly）
· 资源下载页面整理新版本筛选（#3781，by @Chiloven945）
· 帮助库添加“更新 Mod”页面（LTCatt/PCL2Help#361，by @WForst-Breeze）
· 帮助库编修（LTCatt/PCL2Help#355、LTCatt/PCL2Help#364，by @SilianZ、@WorldHim）
· 新主页预设：每日整合包推荐（by @wkea）
· 启动游戏时会在任务栏显示进度条
· 按鼠标侧键可以从其他主页面返回到首页
· 不再给实际上没有包含登录信息的 Access Token 打码
· 将 Java Launch Wrapper 添加到关于的许可与版权声明列表中
· 由于 OptiFabric 已支持 1.14~15，移除 OptiFabric Origins 的安装提示
· 修复了以下 20 个 Bug（包含 2 个严重 Bug）
　　[严重] 安装 OptiFine 时偶尔会失败（#3567，@Herobrine-ZXY）
　　[严重] 启动 Fabric 1.20.5+ 时无法自动选择 JDK21+（#3735，@RainC7）
　　切换第三方登录入口后未要求重新登录（#2313，@mike-brown8，@PAKingdom）
　　左边栏白色背景宽度可能错误（#2982，@liu-zhou-L）
　　在自动更新时手动下载更新不会触发重启（#3044，@3cxc）
　　natives 路径包含特殊字符可能导致崩溃（#3236，@xiaohei20）
　　使用特定 ID 和离线皮肤时无法启动游戏（#3343，@sQianchangyub）
　　错误地显示了 OptiFabric Origins 安装提示（#3403，@QingDa233）
　　使用 Java 8 启动 MC 1.12.2- 时窗口大小有误（#3463，@ZS2147483647）
　　由于 UUID 被打码，启动脚本无法启动游戏（#3612，@mc-124）
　　安装整合包时若压缩包被占用则会跳过解压（#3619，困）
　　部分 Modrinth 支持库被错误地识别（#3725，@Dave12138）
　　部分游戏窗口大小设置选项出现的实际窗口偏大（#3737，@cbs001）
　　从 MC 更新提示点击下载可能导致出错（#3742，@U3L6）
　　安装 OptiFine+Forge 时可能错误地同时释放 JavaWrapper
　　自定义下载的文件名可能有误
　　在特定情况下，Mixin 失败导致的崩溃被误判为其他 Mod 导致崩溃
　　OptiFine 安装器运行失败时可能误判为安装成功
　　下载返回 403 时可能会重试 40 次
　　启动时可能会添加两次 Log4j 参数

<hr /><em>This discussion was created from the release <a href='https://github.com/Hex-Dragon/PCL2/releases/tag/2.7.2'>2.7.2</a>.</em>

**Comment by Deep-Dark-Forest** [comment]
![Screenshot_20240513_183719.jpg](https://github.com/Hex-Dragon/PCL2/assets/131975043/230c8ef0-9600-46b5-a000-fb088f2df722)

龙猫你是不是在发这个discussion的时候特别困啊

**Reply by MoYuan-CN** [reply]
看起来是的（

**Reply by Deep-Dark-Forest** [reply]
@LTCatt

**Reply by LTCatt** [reply]
这个 issue 是转发的，原发现者的 QQ 昵称是困

**Reply by MoYuan-CN** [reply]
啊这……

**Reply by wuliaodexiaoluo** [reply]
啊？

**Comment by RimacC2-EV** [comment]
Hey What？
（若至地问）

![屏幕截图 2024-05-16 194337](https://github.com/Hex-Dragon/PCL2/assets/125170682/1b7ee78f-e5a3-4fc8-b447-0b636e4ef551)

**Reply by Dave-12138** [reply]
重新启动PCL就能更新了<s>，可能是缓存罢</s>

**Reply by MoYuan-CN** [reply]
请注意，2.7.2 是 SnapShot 版更新，Release 版启动器无法接收此更新

**Reply by allMagicNB** [reply]
2.7.2 是快照版更新，正式版能更新才有鬼了……

**Reply by RimacC2-EV** [reply]
<del>瞬间感觉我是真的若至</del>

**Comment by RimacC2-EV** [comment]
> 请注意，2.7.2 是 SnapShot 版更新，Release 版启动器无法接收此更新



好罢，刚才看到Pre-release······<del>感觉我的眼睛不在线</del>
